### ISSUE TITLE
Component arrays rendered alongside siblings render as [object Object]

### ISSUE THREAD

**Issue by spiffytech** [forgojs/forgo#16]
![image](https://user-images.githubusercontent.com/77867/105438838-c7ca2a00-5c31-11eb-9971-56b236c60d6a.png)

If I wrap the array in a parent element (e.g., as the only child inside a `div`) everything renders fine
```tsx
import * as forgo from "forgo";

function Component() {
  return {
    render() {
      const vals = [1, 2, 3];
      return (
        <div>
          <header>Header</header>
          {vals.map((val) => (
            <p>{val}</p>
          ))}
        </div>
      );
    }
  };
}

function ready(fn: any) {
  if (document.readyState !== "loading") {
    fn();
  } else {
    document.addEventListener("DOMContentLoaded", fn);
  }
}
ready(() => {
  forgo.mount(<Component />, document.getElementById("root"));
});

```

[Sandbox](https://codesandbox.io/s/forgo-todos-typescript-forked-mh9hg?file=/src/index.tsx)

**Comment by jeswin** [comment]
Hmm. So what's happening here is, the children is an array containing [<header>, [<p>...]]. 

Forgo right now doesn't know how to render this. I'll fix it right away.

**Comment by jeswin** [comment]
Fixed in v0.0.32.

The children array is flattened before rendering, so there aren't nested arrays (1 level deep) anymore.

**Comment by spiffytech** [comment]
Perfect, thanks! Confirmed working in my project.
